modules/debts_api/app/controllers/debts_api/v0/financial_status_reports_controller.rb

```diff
@@ -345,7 +345,19 @@ def service
       end
 
       def full_transform_service
+        Rails.logger.info(full_transform_logging('info'))
         DebtsApi::V0::FsrFormTransform::FullTransformService.new(full_transform_form)
+      rescue => e
+        Rails.logger.error(full_transform_logging('error'))
+        Rails.logger.error(e.backtrace&.join('\n'))
+
+        raise e
+      end
+
+      def full_transform_logging(type)
+        user_uuid = current_user.uuid
```

Comment:
Is defining user_uuid necessary if it's only called one more time?

Reply:
Probably not, but without this and `submission_id` defined as variables, the final line of this function will be too long and throw an RSpec error. We could probably break the string into multiple lines to avoid this error by doing this:

```ruby
def full_transform_logging(type)
  "DebtsApi::V0::FsrFormTransform::FullTransformService #{type}: " \
    "form ID #{params[:submission_id]} - UUID #{current_user.uuid}"
end
```

cc: @digitaldrk